### PR TITLE
Add capabilities to generate kube

### PR DESCRIPTION
**Pull request by baude**
Using the default capabilities, we can determine which caps were
added and dropped.  Now added them to the security context structure.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: libpod/kube.go [hunk 1]
```diff
@@ -335,20 +336,69 @@ func libpodMountsToKubeVolumeMounts(c *Container) ([]v1.VolumeMount, error) {
...
+	}
+}
+
+func capAddDrop(caps *specs.LinuxCapabilities) (*v1.Capabilities, error) {
+	g, err := generate.New("linux")
+	if err != nil {
+		return nil, err
+	}
+	// Combine all the default capabilities into a slice
+	defaultCaps := append(g.Config.Process.Capabilities.Ambient, g.Config.Process.Capabilities.Bounding...)
```
Comment (mheon) [inline libpod/kube.go]: I'd like @rhatdan to confirm this is the right thing to do - we might just want to choose one of the slices (bounding?)

Comment (giuseppe) [reply libpod/kube.go]: given there is no way to specify what kind of capabilities, it is probably safer to add them all as in the patch

**Comment by giuseppe** [comment]
LGTM

**Comment by rhatdan** [comment]
bot, retest this please
/lgtm

**Comment by rhatdan** [comment]
bot, retest this please

**Pull request merged**